packages/turf-boolean/index.js

```diff
@@ -0,0 +1,21 @@
+var contains = require('./contains');
+var equals = require('./equals');
+var within = require('./within');
+
+/**
+* Contains returns true if the second geometry is completely contained by the first geometry.
+* The contains predicate returns the exact opposite result of the within predicate.
+*
+* @name boolean
+* @param {feature1} feature1
+* @param {feature2} feature2
+* @returns {Boolean}
+* @example
+* var along = turf.contains(line, point);
+*/
+
+module.exports = {
```

Comment:
@DenisCarriere Just wondering what the policy is on namespacing functionality. I thought I recalled reading in some other thread that Turf enforced flat structure and namespacing functions was discouraged. May be mis-remembering.